src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -1196,7 +1196,7 @@ private function assertUrlPackage(ContainerBuilder $container, ChildDefinition $
 
     private function assertVersionStrategy(ContainerBuilder $container, Reference $reference, $version, $format)
     {
-        $versionStrategy = $container->getDefinition((string) $reference);
+        $versionStrategy = $container->getDefinition($reference);
```

Comment:
getDefinition expects a string, not a reference instance. even if we don't enable strict types, it does not mean you need to rely on type conversions.

Reply:
Ok. This type cast were [added on purpose](https://github.com/symfony/symfony/commit/80ed2c5173f2432d59b1c20c9cc1983f490f6c6b#diff-b0c828e732d3d31f0b2bfb2378bd1011) by @nicolas-grekas.
Reverting.